client/my-sites/themes/theme-showcase.jsx

```diff
@@ -58,6 +60,53 @@ const optionShape = PropTypes.shape( {
 	action: PropTypes.func,
 } );
 
+const bannerProps = sample( [
+	{
+		title: translate( 'Are you a photographer? An artist?' ),
+		description: translate(
+			'Explore {{b}}PHOTO BLOG{{/b}}, an elegant theme designed to showcase your visual masterpieces.',
+			{
+				components: {
+					b: <strong />,
+				},
+			}
+		),
+		action: () => {
+			alert( 'Photo Blog banner button clicked!' );
+		},
+		actionLabel: translate( 'See the theme' ),
+		backgroundColor: '#3FE6AF',
+		image: '/calypso/images/themes-banner/photo-blog.png',
+		imageTransform: 'translateY(-4.4%) translateX(17%)',
+		imageAttrs: {
+			alt: translate( 'Photo Blog Theme' ),
+			width: 390,
+		},
+	},
+	{
+		title: translate( 'Do you run a small business?' ),
+		description: translate(
+			'We know you’re crunched for time. We created the quick-setup {{b}}SMALL BUSINESS{{/b}} theme just for you.',
+			{
+				components: {
+					b: <strong />,
+				},
+			}
+		),
+		action: () => {
+			alert( 'Small Business banner button clicked!' );
+		},
+		actionLabel: translate( 'See the theme' ),
```

Comment:
This might be the same for every theme, so we might be able to drop the prop and just hard-wire it in the `ThemesBanner` component.

Reply:
I don't think we will always have themes showcasing in the banners, we may promote a bundle in the future. Not restricting it to themes only I think is good, since we may want to promote other things like a bundle of both a plan and a theme in the future.

Reply:
The thing is called `ThemesBanner` tho 😬 But yeah, okay.

Reply:
Haven't thought about that but you're right, haha!

My initial thought on _Themes_, as in the Theme Team, maybe we should call it PromoBanner, to be more semantic... Any names ring the bell? that's the best one I can think of!